README.md

```diff
@@ -265,10 +279,10 @@ Congrats, you've successfully learnt to
 ## Next Steps
 
 Setting up the tools in your CI/CD pipelines
-- spectral: https://blog.stoplight.io/style-guides-rulebook-series-automated-api-design-checks-in-ci
-- oasdiff: use a GitHub action to check for breaking changes in the CI pipeline: https://www.oasdiff.com/github-action
+- `spectral`: https://blog.stoplight.io/style-guides-rulebook-series-automated-api-design-checks-in-ci
+- `oasdiff`: use a GitHub action to check for breaking changes in the CI pipeline: https://www.oasdiff.com/github-action
 
 
 ## Other Resources
 
-- OpenAPI utilities and demos: https://github.com/CiscoDevNet/OpenAPI_Demo
\ No newline at end of file
+- OpenAPI utilities and demos: https://github.com/CiscoDevNet/OpenAPI_Demo
```

Comment:
Apologies for some whitespace additions. I am unsure what's different in my VSC that would cause this.